### DISCUSSION TITLE
Viewing design ports in hierarchy browser

### DISCUSSION THREAD

**Discussion by gkamendje** [The-OpenROAD-Project/OpenROAD#3584] [Q&A]
Is there a simple way to see design ports information in  hierarchy browser? Right now the only way I found is to go through the inspector and click on the Bterms of top level module. I had hope that in the hierarchy browser, besides leaf instances entries there would be something like ports
```
<top>
 Ports
   Inputs
   Outputs
   IO
 Leaf instances
   Buffer/inverter
   Sequential
   Combinational
 ```

**Comment by maliberty** [comment]
@gadfort any thoughts?

**Reply by gadfort** [reply]
there isn't at the moment, but it should be relatively easy to add to the browser.
